src/App.vue

```diff
@@ -980,6 +984,18 @@ export default class App extends Mixins(
     }
   }
 
+  /** Gets and stores parties info . */
+  private async loadPartiesInformation (): Promise<any> {
+    // NB: will throw if API error
+    let parties = await LegalServices.fetchParties(this.getBusinessId)
+
+    if (parties && parties.parties.length > 0) {
```

Comment:
You should check if `parties.parties` exists.

I recommend conditional chaining to simplify this statement.